webapp/advantage/views.py

```diff
@@ -384,6 +384,26 @@ def get_last_purchase_ids(account_id):
     return flask.jsonify(last_purchase_ids)
 
 
+@advantage_decorator(permission="user", response="json")
+def get_account_users():
+    g.api.set_convert_response(True)
```

Comment:
This doesn't seem to work - `advantage/account-users` returns users in a raw format for me?